handler/additional_types.go.template

```diff
@@ -0,0 +1,22 @@
+// This file is generated with go generate. Any changes to it will be lost after
+// subsequent generates.
+// If you want to edit it go to types.go.template
+
+package handler
+
+import (
+	"github.com/ironsmile/nedomi/config"
+	"github.com/ironsmile/nedomi/types"
+
+    {{range .}}"{{.}}"
+{{end}}
+)
+
+var _ = func() (s struct{}){
```

Comment:
yes there is a reason :smiley:  it doesn't work in any other way. 
I could've done it in `init` but there can only be one `init` function so ... I choose this method. Little bit javascripty if you ask me :laughing: